### ISSUE TITLE
0.17.0-beta.18 changed/broke my arithmetic build

### ISSUE THREAD

**Issue by joeldenning** [jspm/jspm-cli#1900]
Upgrading to beta 18 changed my built file's first line from
```js
define(['exports', 'canopy-urls!sofe', 'lodash', 'sofe'], function (exports, canopyUrls, lodash, sofe) { 'use strict';
```
to
```js
define(['exports', 'canopy-urls!npm:sofe@1.4.0/dist/sofe.js', 'npm:lodash@4.13.1/lodash.js', 'npm:sofe@1.4.0/dist/sofe.js'], function (exports, canopyUrls, npm_lodash_4_13_1_lodash_js, npm_sofe_1_4_0_dist_sofe_js) { 'use strict';
```

Here is my build:
```bash
#!/bin/bash -e

./node_modules/jspm/jspm.js build \
	sentry-error-logging \
	- canopy-urls!sofe \
	- lodash \
	- sofe \
	sofe/sentry-error-logging.js \
	--format amd \
	--source-map-contents \
	"$@"
```

This breaks things for me because the version of lodash that is available in SystemJS is actually `4.11.1` instead of `4.13.1`. In the older built file, it worked fine because it didn't specify a version of lodash, but now that it is I have to have an exact match for that version of lodash.

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/systemjs/builder/commit/a44315bd9555f3525756fc2fe715308b1b96e50c.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.19.

**Comment by joeldenning** [comment]
Thanks!
